source/tools/pascal_pre_proc_Tokyo.dproj

```diff
@@ -1,222 +0,0 @@
-﻿<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">

```

Comment:
Same as above: do not delete Delphi Tokyo files.

Reply:
Same reason here.